src/Node/Stream/StdIO.purs

```diff
@@ -0,0 +1,11 @@
+-- | The standard IO streams for the current process.
+module Node.Stream.StdIO where
```

Comment:
Quite possibly, yes. I was thinking that library (once it exists) could re-export these.

I'm fairly sure that the only way of getting a process object is the global `process` variable (not `require('process')`). The only other similar thing I can think of is a child process, but they are different; for instance, a child process' `stdin`, `stdout`, or `stderr` property might be null.